custom/icds/utils/sms.py

```diff
@@ -0,0 +1,35 @@
+import pytz
+from datetime import timedelta
+
+from corehq.apps.sms.api import get_utcnow
+from corehq.util.timezones.conversions import ServerTime
+
+
+def get_next_available_time(domain_object, domain_now):
```

Comment:
one issue with this is that when the window is reached ALL the messages accumulated over the night will get triggered at exactly the same time.

Reply:
I see, thats a good point. Do you see that will be an issue?

we just get the queued SMSes and queue them for processing again. Celery should be able to handle that.
```    
@classmethod
def get_queued_sms(cls):
    return cls.objects.filter(
        datetime_to_process__lte=datetime.utcnow(),
    )
```